src/Examples/UnitModels/Reactors/plug_flow_reactor_testing.ipynb

```diff
@@ -0,0 +1,1147 @@
+   "metadata": {
+    "tags": [
+     "testing",
+     "remove_cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Check the degrees of freedom\n",
+    "assert degrees_of_freedom(m) == 0"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally, we need to initialize the each unit operation in sequence to solve the flowsheet. In best practice, unit operations are initialized or solved, and outlet properties are propagated to connected inlet streams via arc definitions as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "2021-12-22 13:37:46 [INFO] idaes.init.fs.M101.reagent_feed_state: Starting initialization\n",
+      "2021-12-22 13:37:46 [INFO] idaes.init.fs.M101.reagent_feed_state: Property initialization: optimal - Optimal Solution Found.\n",
+      "2021-12-22 13:37:46 [INFO] idaes.init.fs.M101.catalyst_feed_state: Starting initialization\n",
+      "2021-12-22 13:37:46 [INFO] idaes.init.fs.M101.catalyst_feed_state: Property initialization: optimal - Optimal Solution Found.\n",
+      "2021-12-22 13:37:46 [INFO] idaes.init.fs.M101.mixed_state: Starting initialization\n",
+      "2021-12-22 13:37:46 [INFO] idaes.init.fs.M101.mixed_state: Property initialization: optimal - Optimal Solution Found.\n",
+      "2021-12-22 13:37:46 [INFO] idaes.init.fs.M101.mixed_state: Property package initialization: optimal - Optimal Solution Found.\n",
+      "2021-12-22 13:37:46 [INFO] idaes.init.fs.M101: Initialization Complete: optimal - Optimal Solution Found\n",
+      "2021-12-22 13:37:46 [INFO] idaes.init.fs.H101.control_volume.properties_in: Starting initialization\n",
+      "2021-12-22 13:37:46 [INFO] idaes.init.fs.H101.control_volume.properties_in: Property initialization: optimal - Optimal Solution Found.\n",
+      "2021-12-22 13:37:46 [INFO] idaes.init.fs.H101.control_volume.properties_out: Starting initialization\n",
+      "2021-12-22 13:37:46 [INFO] idaes.init.fs.H101.control_volume.properties_out: Property initialization: optimal - Optimal Solution Found.\n",
+      "2021-12-22 13:37:46 [INFO] idaes.init.fs.H101.control_volume: Initialization Complete\n",
+      "2021-12-22 13:37:47 [INFO] idaes.init.fs.H101: Initialization Complete: optimal - Optimal Solution Found\n",
+      "2021-12-22 13:37:47 [INFO] idaes.init.fs.R101.control_volume.properties: Starting initialization\n",
+      "2021-12-22 13:37:47 [INFO] idaes.init.fs.R101.control_volume.properties: Property initialization: optimal - Optimal Solution Found.\n",
+      "2021-12-22 13:37:47 [INFO] idaes.init.fs.R101.control_volume.reactions: Initialization Complete.\n",
+      "2021-12-22 13:37:47 [INFO] idaes.init.fs.R101.control_volume: Initialization Complete\n",
+      "2021-12-22 13:37:47 [INFO] idaes.init.fs.R101: Initialization Complete: optimal - Optimal Solution Found\n"
```

Comment:
We should add an `InitializationError` (a new exception type) to the property package so that if something breaks in the future we will see exceptions hereto let us know. `InitializationErrors` are currently being added to the unit models in a draft PR in idaes-pse.

Reply:
I will take a look at the PR.